src/azul/service/avro_pfb.py

```diff
@@ -125,9 +131,14 @@ class PFBEntity:
     Python representation of the PFB data object. Attribute names conform to the
     PFB spec (which simplifies serialization).
     """
-    id: str
+
+    def _validate_id(self, attribute, value):
+        require(len(value) <= 254, 'Terra requires IDs be no longer than 254 chars')
```

Comment:
Use `__attrs_post_init__`.

Reply:
```suggestion
    def _validate_id(self, attribute, value):
        reject(len(value) > 254, 'Terra requires IDs be no longer than 254 chars', len(value))
```

Reply:
Instead of logging `len(value)`, I think logging just `value` would be more helpful.